### PR TITLE
March18

### PR DESCRIPTION
**Pull request by samguergen**
@linkblaine @queerviolet

### CODE REVIEW

**Comment by sgharms** [comment]
Keep in mind my comments are _per commit_ ... you may have fixed something later that I saw in a particular commit.  I don't have a better workflow..._yet_.

File: spec/controllers/crud_spec.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  end
+
+  it 'should process the post /post_something route' do
+    # post '/post_something', :params => {info: mypost}
+    post '/post_something'
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('The best')
+    end
```
Comment (queerviolet) [inline spec/controllers/crud_spec.rb:17]: Fix the indentation in this file.

Comment (queerviolet) [inline spec/controllers/crud_spec.rb:17]: And the specs ;)

File: environment.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __FILE__)
+
+require 'sinatra'
+require 'active_record'
+require 'bundler/setup'
+require 'rubygems'
+require 'erb'
+require 'uri'
+require 'pathname'
+require 'erb'
```
Comment (queerviolet) [inline environment.rb:10]: `require 'erb'` is in here twice.

**Pull request closed without merging**